invokeai/frontend/web/src/services/api/endpoints/images.ts

```diff
@@ -15,6 +19,34 @@ export const imagesApi = api.injectEndpoints({
     /**
      * Image Queries
      */
+    listImages: build.query<OffsetPaginatedResults_ImageDTO_, ListImagesArg>({
+      query: (arg) => ({ url: 'images/', params: arg }),
+      providesTags: (result, error, arg) => {
```

Comment:
if we use tags here, then any time the tags are invalidated, bc our query is static, it is re-fetched with the same initial params.

so say you loaded 200 images, and the INITIAL_IMAGE_LIMIT is 100. when the cache is invalidated, it fetches the first 100 images again, and you lose 100 loaded images from the cache. 

we have to manage the listImages cache manually